src/components/ShareTrajectoryButton/index.tsx

```diff
@@ -41,19 +45,25 @@ const ShareTrajectoryButton = ({
             ) : null}
             <Tooltip
                 title={
-                    isDisabled
+                    !simulariumFile.name
                         ? "Load a model to perform this action"
                         : "Share trajectory"
                 }
                 placement="bottomLeft"
                 color={TOOLTIP_COLOR}
                 align={{ offset: tooltipOffset }}
+                mouseEnterDelay={0.5}
+                trigger={isSharing ? [] : ["hover"]}
+                onOpenChange={(visible) => {
+                    setTooltipVisible(visible);
+                }}
```

Comment:
How does `onOpenChange` work when you're also controlling visibility via `open`? Will it still call the callback when the user stops hovering on the tooltip?

Reply:
@ShrimpCryptid  The callback seems to be triggered by the mouse event and not the change in state at least when I played with it and logged it all out.